content/docs/1.6.3/references/helm-values.md

```diff
@@ -160,6 +160,18 @@ You can install Longhorn in an air-gapped environment with a private registry. F
 | privateRegistry.registryUrl | URL of a private registry. When unspecified, Longhorn uses the default system registry. |
 | privateRegistry.registryUser | User account used for authenticating with a private registry. |
 
+### Metrics Settings
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| metrics.serviceMonitor.additionalLabels | object | `{}` | Setting that adds additional labels to the Prometheus ServiceMonitor. |
+| metrics.serviceMonitor.annotations | object | `{}` | Setting that adds annotations to the Prometheus ServiceMonitor. |
```

Comment:
```suggestion
| metrics.serviceMonitor.additionalLabels | `{}` | Additional labels for the Prometheus ServiceMonitor resource. |
| metrics.serviceMonitor.annotations | `{}` | Annotations for the Prometheus ServiceMonitor resource. |
```

Reply:
For brevity, we directly describe the main object and include "setting" only when the supported values are true/false.

Reply:
Note that we cannot remove the `object` type from the table, as it is generated by helm-doc and is present in all tables on this page.

Reply:
Apologies, that was a mistake on my end. I don't know why those cells were removed.